### ISSUE TITLE
Certain values are being halved since upgrading to v3.1.6

### ISSUE THREAD

**Issue by PSXDupe** [oziee/ha-solcast-solar#174]
### The problem

These values have halved.
* Forecast This Hour
* Forecast Next Hour
* Peak Forecast Today
* Peak Forecast Tomorrow

Other values have changed which seen incorrect
* Forecast Today
* Forecast Remaining Today

The values from v3.0.42 match very closely with the data from my Solar Inverter. 

![image](https://github.com/oziee/ha-solcast-solar/assets/36914509/2cd7f50f-0a59-411f-86c6-1a2afacb009f)
![image](https://github.com/oziee/ha-solcast-solar/assets/36914509/1433d8eb-d81d-43db-8eba-c4779e856e9b)

### What version of Solcast Integration has the issue?

v3.1.6

### What version of Home Assistant are you using?

2023.8.4

### Logs debug/info/error output

```txt
N/A
```

**Comment by calvinbui** [comment]
See https://github.com/oziee/ha-solcast-solar/issues/153

tldr: the forecasts are now provided in 30 mins intervals. You want 60 mins (kWh) so multiply by 2.

**Comment by PSXDupe** [comment]
So why call it this hour and next hour when it's not the hourly forecast.

**Comment by campbellcopeland** [comment]
I updated the code from what’s linked here to get old style hourly values back

https://github.com/oziee/ha-solcast-solar/issues/153#issuecomment-1683244435

**Comment by louispires** [comment]
I am also seeing values halved, however mine only happens for historical data:

This screenshot was taken yesterday, which is perfect and accurate:
![WhatsApp Image 2023-08-29 at 21 43 15](https://github.com/oziee/ha-solcast-solar/assets/8841644/c8bba528-a166-4008-a156-528f3c017f23)

Then this was taken this morning for the same day:
![image](https://github.com/oziee/ha-solcast-solar/assets/8841644/aa554fd3-0448-49a0-880b-0c00336512bf)

Thus, the data changed over night and is no longer accurate like it was on the day - All my historical data looks like this now.

**Comment by oziee** [comment]
yes v3.1.x was a flop for some ;) I didn't notice any problems, but some do..
I have completely removed v3.1 from everywhere.. there is a new v4 but its not yet a release, but pre release still
